presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestRoles.java

```diff
@@ -544,6 +544,38 @@ public void testShowRoles()
     }
 
     @Test(groups = {HIVE_CONNECTOR, ROLES, AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testGrantAllRevokeSelect()
+            throws Exception
+    {
+        final String tableName = "default.test_table";
+        onPresto().executeQuery("CREATE ROLE role1");
+        onPresto().executeQuery("GRANT role1 TO alice");
+        onPresto().executeQuery(format("CREATE TABLE %s (foo BIGINT)", tableName));
```

Comment:
Looks like `default.test_table` already exists and hence Travis is failing. Probably some other test already creates this table and doesn't do cleanup?